src/components/contributors-list.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import KeyValueLabel from './key-value-label';
+import _ from 'lodash';
```

Comment:
Should just import `capitalize` here to not include the entire lodash lib in our bundles